scripts/release.py

```diff
@@ -10,7 +10,8 @@
 import ghutils
 from notifications import mail, mailgun, try_context
 
-logger = logging.getLogger("release")
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger()
```

Comment:
Removing the name gives an unhelpful `root` in the log line, `release` is much more descriptive.